CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Command Generate - Fix `mysql2` error with ssl as a boolean.
```

Comment:
[Conventions] This is a technical description of the issue IMO. We should explain to the end users what usage regression has been fixed. Something like:
```suggestion
- Command Generate - Fix projects generation based on MySQL connections. (Regression introduced in v?)
```

Reply:
Thanks!